sympy/sets/handlers/comparison.py

```diff
@@ -23,12 +23,6 @@ def _eval_is_eq(lhs, rhs): # noqa: F811
                lhs.left_open == rhs.left_open,
                lhs.right_open == rhs.right_open)
 
-
-@dispatch(FiniteSet, Interval) # type:ignore
-def _eval_is_eq(lhs, rhs): # noqa: F811
-    return False
```

Comment:
Why is this removed?

Reply:
It was duplicate.This same is written twice in codebase.